templates/rhdm71-authoring-ha.yaml

```diff
@@ -0,0 +1,1216 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  annotations:
+    iconClass: icon-jboss
+    tags: rhdm,jboss,authoring,ha
+    version: "1.0"
+    openshift.io/display-name: Red Hat Decision Manager 7.1 authoring environment (HA, persistent, with https)
+    openshift.io/provider-display-name: Red Hat, Inc.
+    description: Application template for a HA persistent authoring environment, for Red Hat Decision Manager 7.1
+    template.openshift.io/long-description: This template defines resources needed to develop and execute Rules and Poocesses on Red Hat Decision Manager 7.1 with High Availability enabled as tech preview, including application deployment configuration, secure and insecure http communication and persistent volume to store data that needs to survive to a restart.
+    template.openshift.io/documentation-url: https://access.redhat.com/documentation/en-us/red_hat_decision_manager/7.1/html/deploying_a_red_hat_decision_manager_7.1_authoring_environment_on_red_hat_openshift_container_platform/
+    template.openshift.io/support-url: https://access.redhat.com
+    template.openshift.io/bindable: "false"
+  name: rhdm71-authoring-ha
+labels:
+  template: rhdm71-authoring-ha
+  rhdm: "1.0"
+message: |-
+          A new persistent HA Decision Manager application have been created in your project.
+          The username/password for accessing the Decision Central interface is
+
+              Username: ${KIE_ADMIN_USER}
+              Password: ${KIE_ADMIN_PWD}
+
+          The user name/password for calls to the Execution Server is
```

Comment:
As per this https://github.com/jboss-container-images/rhdm-7-openshift-image/pull/106 everything was renamed to Kie Server, shouldn't these new templates have the same name?
(comment valid for all Execution server in the proposed templates)

Reply:
Yup; already handled.